package-lock.json

```diff
@@ -25529,6 +25648,23 @@
         "rgb2hex": "^0.2.0",
         "serialize-error": "^7.0.0",
         "webdriver": "6.1.17"
+      },
+      "dependencies": {
+        "serialize-error": {
+          "version": "7.0.1",
```

Comment:
This is a duplicate now. Let's `npm dedupe` this and see what we can fix and what we can resolve manually